crates/tools/src/main.rs

```diff
@@ -229,6 +230,84 @@ fn main() {
                         .takes_value(true)
                         .default_value("http://127.0.0.1:8119")
                         .help("Godwoken jsonrpc rpc sever URL"),
+                )
+                .arg(
```

Comment:
IMO the cli should automatically try `gw_` version RPC after we failed to request a RPC.

```
Failed to request /xxx
Retry /gw_xxx ...
```

Reply:
Updated.